tests/test_data/endpoints.py

```diff
@@ -0,0 +1,5 @@
+favorites_url = 'https://my.prom.ua/cabinet/user/favorites'
+velosipednye_shiny_url = 'https://prom.ua/Velosipednye-shiny'
+
```

Comment:
both url have the common part, so it'd be nice to move it out as a separate part and then use it with string transpolation like:
prom_host = 'https://my.prom.ua/'
fav_url = f'{prom_host}/cabinet/user/favorites'
and etc.
It will save you time if the endpoints number increases or it could become necessary to start testing not only in Prod, but using other environments.

Reply:
Urls are removed to the class PromEndpoints